internal/pkg/cli/env_init.go

```diff
@@ -300,6 +327,85 @@ func (o *initEnvOpts) askCustomizedResources() error {
 	if o.NoCustomResources {
 		return nil
 	}
+	useDefaultEnv, err := o.prompt.Confirm(envInitDefaultEnvConfirmPrompt, "")
```

Comment:
I wonder if instead of a `Confirm` this should be a `Select`?
```
Would you like to use the default configuration for a new production environment?
    - blah blah

> Yes, use default.
> Yes, but I'd like configure the default resources (CIDR ranges).
> No, I'd like to import existing resources (VPC, subnets).
```